.env.example

```
@@ -1,6 +1,6 @@
-VITE_API_KEY="your-firebase-api-key"
-VITE_AUTH_DOMAIN="your-firebase-auth-domain"
-VITE_PROJECT_ID="your-firebase-project-id"
-VITE_STORAGE_BUCKET="your-firebase-storage-bucket"
-VITE_MESSAGING_SENDER_ID="your-firebase-messaging-sender-id"
-VITE_APP_ID="your-firebase-app-id"
+VITE_FIREBASE_API_KEY= 
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)